example/todo-example/cypress.json

```diff
@@ -1,3 +1,6 @@
 {
-    "video": false
+    "video": false,
+    "env": {
+        "headersBlocklist": ["ignore-me-global"]
```

Comment:
possibly change this to `ignore-me-globally` to match up with the example in the readme, or change the readme to `ignore-me-global` as that is used else in your example code 👍🏾

Reply:
Good cal, consistency is the key!